asciidoc/modules/con_mg_changes-in-vertx-service-discovery.adoc

```diff
@@ -17,3 +17,19 @@ The following create methods in service discovery that have `Handler<ServiceDisc
 == Service importer and exporter methods are no longer fluent
 
 The `ServiceDiscovery.registerServiceImporter()` and `ServiceDiscovery.registerServiceExporter()` methods are no longer fluent. The methods return `Future<Void>`.
+
+
+== Kubernetes Service Importer no longer registered automatically
+
+Since {VertX} {v4}, the Kubernetes bridge is no longer registered automatically.
+It is not sufficient anymore to have it on the _classpath_.
+
+It must be registered manually after creating `ServiceDiscovery`:
+
+[source,java,options="nowrap",subs="attributes+"]
+----
+JsonObject defaultConf = new JsonObject();
+serviceDiscovery.registerServiceImporter(new KubernetesServiceImporter(), defaultConf);
+----
+
```

Comment:
@tsegismont Would a rephrase as below work?

== Kubernetes service importer is no longer registered automatically

The `vertx-service-discovery-bridge-kubernetes` adds the KubernetesServiceImporter discovery bridge. The bridge imports services from Kubernetes or Openshift into the Eclipse Vert.x service discovery.

From {VertX} {v4}, this bridge is no longer registered automatically. Even if you have added the bridge in the classpath of your Maven project, it will not be automatically registered.

You must manually register the bridge after creating the `ServiceDiscovery` instance.

The following example shows you how to manually register the bridge.

[source,java,options="nowrap",subs="attributes+"]
----
JsonObject defaultConf = new JsonObject();
serviceDiscovery.registerServiceImporter(new KubernetesServiceImporter(), defaultConf);
----

Reply:
LGTM @sangeetaraghu . How do you want to proceed? Should I copy/paste this in the PR?

Reply:
@tsegismont If you can copy/paste, that would be great! Easiest option :D

Reply:
Done